### DISCUSSION TITLE
Build gets stuck at queued for more than 10 mins

### DISCUSSION THREAD

**Discussion by samnbd** [codemagic-ci-cd/codemagic-docs#2388] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Start generating a build. 

### Expected results

Expected for the build to start actually building.

### Actual results

The build get stuck at queued for more than 10 mins now.
Tried to stop and start new build but it's the same thing...

### Build id (optional)

65d4e5e1b6d9e380252610f8

**Comment by MehmetKaranlik** [comment]
Same here

Build ID : 65d4e8531791492c2fd0c2c7

**Comment by Johan-glitch1412** [comment]
Experiencing the same issue on our side with flutter builds as well.

**Comment by mtebele** [comment]
Same here trying to build React Native.

**Reply by mtebele** [reply]
Both Android and iOS

**Comment by irby** [comment]
Same here with my React Native iOS builds.

**Comment by MattyBoy4444** [comment]
Same
Build ID: 65d4e9e6cd2940e35d4ae4d9

**Comment by irby** [comment]
Similar discussion from last year when builds were stuck:
https://github.com/orgs/codemagic-ci-cd/discussions/1482

**Comment by mikhail-tokarev** [comment]
We are currently investigating why builds are not getting started. I'll update this thread once we have more information.

**Reply by mikhail-tokarev** [reply]
We have confirmed the outage and currently investigating to determine the root cause.

**Reply by mikhail-tokarev** [reply]
We've applied a mitigation to fix the issue with the latest deployment. It will be resolved within 5-10 minutes. Thanks for your patience!

**Reply by irby** [reply]
Thank you for the speedy response!

**Reply by abarone-btf** [reply]
@mikhail-tokarev we are still seeing queued tasks :/. Is it normal?, or its just a matter of time?

**Reply by mikhail-tokarev** [reply]
Unfortunately we still need more time to resolve the issue :(

**Reply by irby** [reply]
Confirmed that my builds are still in queue. I've tried cancelling all queued items and starting new builds but did not fix it for me.
Build ID: 65d4fa94483cbccb19cbc826

**Reply by mikhail-tokarev** [reply, marked as answer]
The incident has been resolved, but it may still take some time to resolve the build queue and start all builds. We apologize for the inconvenience caused!

**Reply by wilsolutions** [reply]
I have just ran at the same issue, I don't want to run the previous build, I just want to be able to start a new one.
Please, any update?
This is the build id: 65d4ffeaf859b10daecaa008

**Reply by irby** [reply]
My queued build just executed. 🎉

**Reply by wilsolutions** [reply]
Building here again as well!
Thanks, have a good one everyone!

**Reply by samnbd** [reply]
Thanks for fixing this

**Comment by Sophist-UK** [comment]
Same here on a platformio build job. I cancelled the old job that had been waiting for more than a day and submitted a new (different) job.

**Reply by Sophist-UK** [reply]
New job has now been queued for 10 hours. @mikhail-tokarev What is happening please?

**Reply by helinanever** [reply]
Hi @Sophist-UK , can you please try canceling that build as well and start a new one? Yesterday's issue has been resolved and your new build is expected to start quickly.

**Reply by Sophist-UK** [reply]
@helinanever Sorry - it turned out to be user error. D'oh.
